Upload: fix missing version in metadata.json (#3111)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased

> * Fixed an issue in **upload** where the installed version was missing from the XSOAR and XSIAM UI.

Reviewer: ```suggestion
* Fixed an issue where **upload** would not properly show the installed version in the UI.
```
